### DISCUSSION TITLE
Deprecate Doctrine Annotation rules

### DISCUSSION THREAD

**Discussion by julienfalque** [PHP-CS-Fixer/PHP-CS-Fixer#6964] [RFC]
I think we can safely assume Doctrine Annotation usage will decline in favor of PHP 8 attributes. I propose to deprecate all rules and the ruleset related to Doctrine Annotations in next 2.x release and remove them 3.x.

**Comment by Wirone** [comment]
FYI: in #7251 I've prepared changes that would make Doctrine's packages optional, we then could extract those fixers to separate package and mark it as abandoned, so users could use it when needed, but we wouldn't be forced to maintain them further.
